src/pages/warp.tsx

```diff
@@ -4,6 +4,30 @@ import Marquee from 'react-fast-marquee'
 import Link from 'next/link'
 import Image from 'next/image'
 
+function showPopup1() {
+  const popup = document.getElementById('popup1');
+  popup.classList.add('active');
+}
+function hidePopup1() {
+  const popup = document.getElementById('popup1');
+  popup.classList.remove('active');
+}
+function showPopup2() {
+  const popup = document.getElementById('popup2');
+  popup.classList.add('active');
+}
+function hidePopup2() {
+  const popup = document.getElementById('popup2');
+  popup.classList.remove('active');
+}
+function showPopup3() {
+  const popup = document.getElementById('popup3');
+  popup.classList.add('active');
+}
+function hidePopup3() {
+  const popup = document.getElementById('popup3');
+  popup.classList.remove('active');
+}
```

Comment:
These need better names. I'm also not sure why all of these are a seperate function...